src/EditorFeatures/CSharpTest/Completion/CompletionProviders/XmlDocumentationCommentCompletionProviderTests.cs

```diff
@@ -1010,25 +1079,6 @@ public class foo<T>
             await VerifyCustomCommitProviderAsync(markupBeforeCommit, "summary", expectedCodeAfterCommit, commitChar: '>');
         }
 
-        [WorkItem(623168, "http://vstfdevdiv:8080/DevDiv2/DevDiv/_workitems/edit/623168")]
-        [WpfFact, Trait(Traits.Feature, Traits.Features.Completion)]
-        public async Task NoTrailingSpace()
-        {
-            var markupBeforeCommit = @"class c
-{
-        /// $$
-        void foo() { }
-}";
-
-            var expectedCodeAfterCommit = @"class c
-{
-        /// <see cref=""$$""/>
```

Comment:
❗️ I rely on the existing behavior frequently (e.g. search for `see cref` in [CoreTaskExtensions.cs](https://github.com/tunnelvisionlabs/dotnet-threading/blob/master/Rackspace.Threading/CoreTaskExtensions.cs)). I am sure I'm not the only one who will feel great pain at this regression.